src/toUinteger32.js

```diff
@@ -0,0 +1,27 @@
+import { curryN } from 'ramda';
+
+/**
+ * Converts the value to an unsigned 32-bit integer number
+ *
+ * @func toUinteger32
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.28.0|v2.28.0}
+ * @category Math
+ * @sig * -> Number
+ * @param {*} val Value to be converted
+ * @return {Number}
+ * @example
+ *
+ * RA.toUinteger32(1.5); //=> 1
```

Comment:
Let's model the example more about is the important thing that this function does.
```js
RA.toInteger32(2 ** 35); // => 0
RA.toInteger32(2 ** 31); // => 2147483648
RA.toInteger32(2 ** 30); // => 1073741824